packages/core/src/StateNode.ts

```diff
@@ -1893,7 +1901,7 @@ class StateNode<
     const delayedTransitions = this.after;
 
     const formattedTransitions = flatten(
-      [...doneConfig, ...invokeConfig, ...onConfig].map(
+      [...doneConfig, ...invokeConfig, ...onConfig, ...eventlessConfig].map(
```

Comment:
maybe this should be put before `onConfig` to take precedence over `''`?

---

this is definitely a quick way for doing this and is good for now, but I feel that in the future we should check if maybe moving those to a separate property would be preferable (maybe not)

Reply:
There is some complicated logic that needs to be fixed in V5 if `eventlessConfig` comes before `onConfig` - if you try it, tests won't pass if you have it the other way around.

Reply:
Hm, interesting - I would not have expected it to matter 🤔 Might take a look later at this.